src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1981,8 +1981,16 @@ def get_multisite_status(self):
             is_multisite_configured = False
         return is_multisite_configured
 
-    def get_multisite_sync_status(self):
+    def get_multisite_sync_status(self, daemon_name: str):
+        if daemon_name:
+            orch = OrchClient.instance()
+            daemons = [d.to_dict() for d in orch.services.list_daemons()]
+            rgw_daemons = [d for d in daemons if d['daemon_type'] == 'rgw']  # noqa E501 #pylint: disable=line-too-long
```

Comment:
```suggestion
            rgw_daemons = [d for d in daemons if d['daemon_type'] == 'rgw']  
```
Is the line actually too long here?